src/components/Button/Button.tsx

```diff
@@ -71,8 +71,7 @@ export const useStyles = makeStyles(
         boxShadow: theme.boxShadows.focusVisibleInverse,
       },
       '&:disabled, &[disabled]': {
-        backgroundColor: 'rgba(255, 255, 255, 0.45)',
-        opacity: 1,
+        backgroundColor: theme.hexToRgba(theme.palette.common.white, 0.45),
```

Comment:
Were these intentional in this PR?

Reply:
Hmm... no, I don't think so. That may be an artifact of my rebase. Looking into it

Reply:
@xdumaine @HaleyWardo OK, fixed it!